src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+        "parentCode": "ITEM"
+      }
+    }
+  ]
+}
+````
+
+Example 3:
+```json
+{
+  "status": "PARTIAL_SUCCESS",
+  "recordsSucceeded": 2,
+  "recordsFailed": 1,
+  "errors": [
+    {
+      "message": "This item code is already used by another item in the same list",
+      "listItem": {
+        "shortCode": "CHILD_ITEM_ONE",
+        "value": "CHILD_ITEM_ONE",
+        "parentCode": "ITEM"
+      }
+    }
+  ]
+}
+```
+## <a name="patch-list-item-bulk"></a>Update List Items
+
+Update list items by long code for a specific list with provided request body.
+
+### Scopes
+
+`spend.listitem.write` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+#### URI
+
+##### Template
+
+```shell
+PATCH /list/v4/lists/{listId}/bulk
+```
+
+##### Parameters
+
+|Name|Type|Description|
+|---|---|---|
+|`Accept-Language`|`string`|Language code. Default: `en` |
+|`company-uuid`|`string`|Company UUID of the user.|
+|`concur-correlationid`|`string`|The unique identifier of the consumer making the API calls. Minimum length: 6 characters|
+|`concur-uuid`|`string`|SAP Concur employee UUID of the caller.|
+|`listId`|-|**Required** The unique identifier of the list (SYNC_GUID).|
+
+#### Headers
+
+* [RFC 7231 Accept](https://tools.ietf.org/html/rfc7231#section-5.3.2)
+* [RFC 7231 Accept-Language](https://tools.ietf.org/html/rfc7231#section-5.3.5)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+
+#### Payload
+
+* [Bulk List Item Update Request](#schema-bulk-list-item-update-request)
+* [Bulk List Item Update Request Part](#schema-bulk-list-item-update-request-part)
+
+### Response
+
+#### Status Codes
+
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)
+* [206 Partial Content](https://www.rfc-editor.org/rfc/rfc7233#section-4.1)
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)
+* [401 Unauthorized](https://tools.ietf.org/html/rfc7235#section-3.1)
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4)
+* [415 Unsupported Media Type](https://tools.ietf.org/html/rfc7231#section-6.5.13)
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1)
+
+#### Headers
+
+* `concur-correlationid` is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+* [RFC 7230 Content-Length](https://tools.ietf.org/html/rfc7230#section-3.3.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Date](https://tools.ietf.org/html/rfc7231#section-7.1.1.2)
+* [RFC 7234 Cache-Control](https://tools.ietf.org/html/rfc7234#section-5.2)
+
+
+#### Payload
+
+* [Bulk List Item Response](#schema-bulk-list-item-response)
+* [Bulk List Error](#schema-bulk-list-error)
```

Comment:
Same comment - This shouldn't be needed since the "Bulk List Item Response" is the response object.
```suggestion
```